tests/apps_test.go

```diff
@@ -204,4 +207,68 @@ var _ = Describe("Apps", func() {
 			login(url, testUser, testPassword)
 		})
 	})
+
+	Context("with a custom buildpack deployed app", func() {
+		var cleanup bool
+		var testApp App
+
+		BeforeEach(func() {
+			cleanup = true
+			os.Chdir("example-perl")
+			appName := getRandAppName()
+			createApp(appName, "--buildpack", "https://github.com/miyagawa/heroku-buildpack-perl.git")
+			testApp = deployApp(appName)
+		})
+
+		AfterEach(func() {
+			defer os.Chdir("..")
+			if cleanup {
+				destroyApp(testApp)
+			}
+		})
+
+		It("can get app info", func() {
+			sess, err := start("deis info -a %s", testApp.Name)
```

Comment:
My only suggestion would be to create helper functions containing the logic in `"can get app info"` and the logic in `"can open the app's URL"`, since we are now re-using it in multiple Contexts in this test.

Perhaps something like `verifyAppInfo(testApp app)` and `verifyAppOpen(testApp app)`